NPM packages misses "default" exports

### Description

The published package misses the `default: "./dist/index.js"` exports property, which prevents users to import the library.

### Version

1.1.0-alpha.9

### Steps to Reproduce

1. Create a test project with Jest and TSFlow
2. Define tests
3. Run Jest

### Logs

```shell
FAIL  src/__tests__/calculator.test.ts
  ● Test suite failed to run

    Cannot find module '@baento/tsflow' from 'src/__tests__/calculator.test.ts'

    > 1 | import { loadFeature } from "@baento/tsflow";
        | ^
      2 |
      3 | import "./calculator.steps";
      4 | import "./result.steps";

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (src/__tests__/calculator.test.ts:1:1)

------|---------|----------|---------|---------|-------------------
File  | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
------|---------|----------|---------|---------|-------------------
...es |       0 |      100 |       0 |       0 |                   
 ...s |       0 |      100 |       0 |       0 | 1-9               
------|---------|----------|---------|---------|-------------------
Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        3.266 s
```


### Operating System

Windows 11

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow TSFlow's Code of Conduct

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "license": "MIT",
   "type": "module",
   "exports": {
+    "default": "./dist/index.js",
     "import": "./dist/index.js",
     "types": "./dist/index.d.ts"
   },

